e2e-tests/portal/cypress/e2e/journeys/dashboard/dashboard-deals/filters/dashboard-deals-filters.spec.js

```diff
@@ -152,6 +152,11 @@ context('Dashboard Deals filters', () => {
       dashboardDeals.filters.panel.form.status.rejectedByUKEF.checkbox().should('exist');
       dashboardDeals.filters.panel.form.status.rejectedByUKEF.checkbox().should('not.be.checked');
 
+      // cancelled
+      dashboardDeals.filters.panel.form.status.cancelled.label().contains(CONSTANTS.DEALS.DEAL_STATUS.CANCELLED);
+      dashboardDeals.filters.panel.form.status.cancelled.checkbox().should('exist');
+      dashboardDeals.filters.panel.form.status.cancelled.checkbox().should('not.be.checked');
```

Comment:
If this is behind a FF we might need to re-write it :(

Reply:
Great points, have redone the E2E tests